src/system.ts

```diff
@@ -182,6 +183,23 @@ export const gqlQueryable: GqlQueryable = (() => {
   return db;
 })();
 
+export type PostgreSQL = postgres.Sql;
+
+export let psql: PostgreSQL | undefined = undefined;
```

Comment:
Let's call this `legacyPsql` and prefix all the environment variables with `LEGACY_`. It's not imminent, but I suspect we'll add "native" Postgres support at some point. When we do we'll want to use different environment variables for that.

Reply:
fixed